examples/netlify-functions/functions/addToCart/addToCart.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { Cart } = require('../../models');
+
+const handler = async(event) => {
+  try {
+    if (event.body.cartId) {
+      // get the document containing the specified cartId
+      const cart = await Cart.findOne({ _id: event.body.cartId });
+      cart.products.push(event.body.product);
```

Comment:
Check for duplicates. We only want to have at most one entry with a given `productId` in the cart.